### DISCUSSION TITLE
DDR3 1333 vs 1600 vs 1866 MHz with PCIe 3 16x

### DISCUSSION THREAD

**Discussion by hajes** [madMAx43v3r/chia-gigahorse#97] [General]
I already see PCIe slot maxed out at 12-15GBs with 1333MHz DDR3 ECC RAM. I am getting ~3.11 (187s), which similar to Max test machine with 1600MHz DDR3.

It makes me wonder if there is a point to buy 1866MHz DDR3 RAM. I guess it won't plot much faster...perhaps 160s for 600$ price tag :-D

Buying second GPU be may be better resource allocation despite both PCIe slots are connected to CPU2, and there will be most likely speed penalty. Both GPUs hogging CPU2 <> PCIe <> QPI link to CPU1/RAM

**Comment by madMAx43v3r** [comment]
> I am getting ~3.11 (187s), which similar to Max test machine with 1600MHz DDR3.

With latest version I'm getting 145 sec for C8. You are def RAM bound, no point in adding another GPU.

**Comment by hajes** [comment]
1877MHz RAM produces 172s (2.9min) plots if kept under 60°C ~7% faster
